resources/templates/apache2/vhost.template

```diff
@@ -26,8 +26,8 @@
             # See: https://wiki.apache.org/httpd/PHP-FPM#apache_httpd_2.4
 
             # For best performance, prefer socket use. This requires Linux, and that both Apache and PHP has access to
-            # the socket file `/var/run/php5-fpm.sock` via local file system and hence run on the same machine.
-            #SetHandler "proxy:unix:/var/run/php5-fpm.sock|fcgi://localhost/"
+            # the socket file `/var/run/php/php8.3-fpm.sock` via local file system and hence run on the same machine.
+            #SetHandler "proxy:unix:/var/run/php/php8.3-fpm.sock|fcgi://localhost/"
```

Comment:
```suggestion
            #SetHandler "proxy:unix:/var/run/php/php-fpm.sock|fcgi://localhost/"
```
everywhere and you're gonna make this future proof without worrying about PHP 10 (internal pun) ;)

Reply:
@mateuszbieniek does this suggestion work for you? Don't want to change after you've already approved 😉

Reply:
oh yeah, just checked and there is indeed php-fpm.sock defaulting to 8.3 in my instance!

Reply:
oh boy, 20 years of using Linux and did not notice that...

Reply:
Improved in https://github.com/ibexa/post-install/pull/87/commits/9d966e666d2e3fe2e9d834b9f145b15a47c958b7 , thank you!